pymatviz/ptable.py

```diff
@@ -235,8 +239,8 @@ def data_preprocessor(data: SupportedDataType) -> pd.DataFrame:
         OR
         >>> data_df: pd.DataFrame = pd.DataFrame(
             data_dict.items(),
-            columns=["Element", "Value"]
-            ).set_index("Element")
+            columns=[{element_col}, {value_col}]
```

Comment:
It looks like this breaks the site builder workflow. I should have used a raw string instead. Please help me fix this. Thanks!

Reply:
will fix, PR incoming for SSOT of keys across the whole code base